katago_server/trainings/serializers/network.py

```diff
@@ -22,14 +22,36 @@ class Meta:
             "model_file",
             "model_file_bytes",
             "model_file_sha256",
+            "model_zip_file",
             "parent_network",
+            "log_gamma",
+            "log_gamma_uncertainty",
+            "log_gamma_lower_confidence",
+            "log_gamma_upper_confidence",
         ]
         extra_kwargs = {
             "url": {"lookup_field": "name"},
             "run": {"lookup_field": "name"},
             "parent_network": {"lookup_field": "name"}
         }
 
+    def create(self, validated_data):
+        data = validated_data.copy()
+        if "parent_network" in data:
+            if data["parent_network"]:
+                data["log_gamma"] = data["parent_network"].log_gamma
+                data["log_gamma_uncertainty"] = 2.0
```

Comment:
That is to create with a given (big enough) uncertainty and the parent elo, it is probably closer to reality.